apps/domain/src/main/routes/users/routes.py

```diff
@@ -1,37 +1,79 @@
-from .blueprint import users_blueprint as user_route
-from flask import request, Response
 import json
 
-from syft.grid.messages.user_messages import (
-    CreateUserMessage,
-    DeleteUserMessage,
-    GetUserMessage,
-    GetUsersMessage,
-    UpdateUserMessage,
-    SearchUsersMessage,
-)
+from flask import Response, request
+from syft.grid.messages.user_messages import (CreateUserMessage,
+                                              DeleteUserMessage,
+                                              GetUserMessage, GetUsersMessage,
+                                              SearchUsersMessage,
+                                              UpdateUserMessage)
 
-from ..auth import error_handler, token_required, optional_token
-from ...core.task_handler import route_logic, task_handler
-from ...core.node import get_node
-from ...core.exceptions import MissingRequestKeyError
 from ...core.database.utils import model_to_json
+from ...core.exceptions import MissingRequestKeyError
+from ...core.node import get_node
+from ...core.task_handler import route_logic, task_handler
+from ..auth import error_handler, optional_token, token_required
+from .blueprint import users_blueprint as user_route
 
 
 @user_route.route("me", methods=["GET"])
 @token_required
 def me(current_user):
     user = model_to_json(current_user)
-    return Response(
-        json.dumps(user),
-        status=200,
-        mimetype="application/json",
-    )
+    return Response(json.dumps(user), status=200, mimetype="application/json")
 
 
 @user_route.route("", methods=["POST"])
 @optional_token
-def create_user(current_user):
+def create_user(current_user):  # noqa
+    """Create user.
+
+    First user created will be administrator by default.
+    ---
+    operationID: createUser
+    requestBody:
+        description: Add user
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - email
+                - password
+              properties:
+                email:
+                  type: string
+                password:
+                  type: string
+                token:
```

Comment:
The user creation only requires `username` and `password`, the `token` is needed only when calling `POST /setup`

Reply:
I though that was optional (at least that is what I can infer from the code), meaning that if you pass the token you could create some especial users such as adminsitrators. see: https://github.com/OpenMined/PyGrid/blob/0a042991da2e77048068cf1f1bd0ca26d0d0995c/apps/domain/src/main/routes/users/routes.py#L37